src/Groups/GAPGroups.jl

```diff
@@ -590,7 +609,7 @@ where `H^z=K`; otherwise, return
 false, nothing
 ```
 """
-function isconjugate(G::GAPGroup, H::GAPGroup, K::GAPGroup)
+function representative_action(G::GAPGroup, H::GAPGroup, K::GAPGroup)
    conj = GAP.Globals.RepresentativeAction(G.X, H.X, K.X)
    if conj != GAP.Globals.fail
       return true, group_element(G, conj)
```

Comment:
aside: I wonder if in case of failure we should return `false, one(G)` for the sake of type stability?

Reply:
For me it is fine. We used `false, nothing` in many other occurrences, though.

Reply:
Yeah, I wonder this for those, too... Anyway, this PR is not the time and place to change it, I just wonder.

Reply:
Of course if we keep it at "nothing" then we can change the function by dropping the boolean part of the return value; given its name, I'd expect only the group element to be returned (or nothing...)

Reply:
Indeed, in GAP only one value is returned, while the `representative_action` already defined in gsets.jl return the boolean value too. Do I drop the boolean output or do we keep it for the moment?

Reply:
We should match what the other `representative_action` methods do right now. So if we they return two values, then we should do it here, too, at least for now -- and if we want to change it, change it simultaneously in all methods.